subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -377,6 +377,19 @@ config BT_CTLR_SCAN_ENABLE_STRICT
 	  Enforce returning HCI Error Command Disallowed on enabling/disabling
 	  already enabled/disabled scanning.
 
+config BT_CTLR_ISOAL_SN_STRICT
+	bool "Enforce Strict Tx ISO Data Sequence Number use"
```

Comment:
```suggestion
config BT_CTLR_ISOAL_PSN_STRICT
	bool "Enforce Strict Tx ISO Data Packet Sequence Number use"
```

Reply:
Hmm, I see that the controller actually uses `sn` multiple places for this value, so perhaps we should just keep it as `SN`/`Sequence Number` rather than the full name of the field (Packet Sequence Number).

Reply:
Keeping "sequence number" phrase as ISOAL specification uses this phrase (and does not use "packet sequence number"). `Packet_Sequence_Number` is only mentioned in the context of HCI ISO Data packets.

Reply:
Correct, but the sequence number in ISOAL is the 1-bit value that is either 0 or 1, right? 

This Kconfig only applies to the packet sequence number that the host provides when sending over HCI, right? 

I am OK with using use sequence number

Reply:
> but the sequence number in ISOAL is the 1-bit value that is either 0 or 1, right?

This is a bit confusing in Spec, in ISOAL SDU sequence number is 16-bit to match the value `Packet_Sequence_Number` in HCI ISO Data packet. The other `SN` and `NESN` these are in the PDU header.

```
The sequence number shall be incremented for each SDU_Interval, whether or
not an SDU was received from the upper layer or included in reports sent to the
upper layer.
```

The Spec uses the term `upper layer`, hence is open to having other interfaces (not just HCI as the interface) for sending/receiving Tx ISO Data packets.

Reply:
Okay so "sequence number" in ISOAL refer to any sequence number from upper layers (does that include vendor specific data paths?), and "packet sequence number" is only used for HCI. 

I'd argue that if this check is only for HCI ISO data packets, then it makes sense to use the term "packet sequence number", but if it is for any upper layer, then stick with "sequence number".